client/components/plans/plan-list/index.jsx

```diff
@@ -78,6 +79,10 @@ const PlanList = React.createClass( {
 			const filteredPlans = filterPlansBySiteAndProps( plans, site, hideFreePlan, showJetpackFreePlan );
 
 			plansList = filteredPlans.map( plan => {
+				if ( ! isEnabled( 'upgrades/personal-plan' ) && plan.product_slug === 'personal-bundle' ) {
```

Comment:
I just wanted to mention it, I see what you mean :).
Bummer we have to introduce bunch of new plans for different price points :/
Maybe we could save test variant in `/me/settings` and return different price points based on that?